### ISSUE TITLE
file missing for training stlt

### ISSUE THREAD

**Issue by patrolli** [gorjanradevski/revisiting-spatial-temporal-layouts#1]
hello, I tried to train the stlt model but got some errors. I first run create_dataset.py to prepare dataset, after that, I canot find the videoid2size.json file, which however is necessary for running the code. How can I get this json file? Thanks a lot ~

**Comment by gorjanradevski** [comment]
Good catch. You can download the .json file from this link: https://drive.google.com/file/d/1ANaDAxXoA63CA9zXalnmaskqfO4cftW4/view?usp=sharing

Let me know if you encounter any other issues!

**Comment by patrolli** [comment]
Thanks! I encountered another problem but not sure is caused by this repo. When I finish running the create_dataset.py code, the dump size of the result json file is 0, as shown in figure. 
![image](https://user-images.githubusercontent.com/25717445/154481600-a4955112-ccb5-4ed5-926c-2832306d1b74.png)
I am not sure what is caused by, could you please provide the resutl json file? Thanks a lot ~

**Comment by gorjanradevski** [comment]
I'm closing this issue, feel free to reopen if the problem persists.
